.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: Test
+on:
+  pull_request: {}
+  push:
+    branches:
+    - master
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        node-version: ['10', '12', '14']
+    name: Node.js ${{ matrix.node-version }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npx lerna bootstrap
```

Comment:
FYI I heard that this NPM 7 now can manage monorepo: https://github.blog/2020-10-13-presenting-v7-0-0-of-the-npm-cli/

Reply:
It looks like https://github.com/npm/rfcs/blob/latest/implemented/0026-workspaces.md is the feature in question.

It looks like that'll ship with Node.js 15, although it is possible to `npm install -g npm@7` and get it on older Nodes.js, I did it with Node.js 12.

@dontcallmedom would you be OK with requiring npm 7?

Reply:
I'm having some trouble getting workspaces to work, and I wonder if it really has the features we want. In particular, I can't find an equivalent to `lerna publish`, and I'm not sure if `npm publish` will support publishing a new version only if there are changes.

So I'm leaning towards trying Lerna first, and migrating later if others using Lerna successfully do so.

Reply:
https://github.com/w3c/webref/commit/faab9f616a8b8b230e58b4066c0190aa49c1a398 is what I tried.

Reply:
using the default (even if new) npm stuff would have been my preference, but it's still immature or an ill-fit, Lerna is what @tidoust and I had discussed when we had looked into this. In particular, I assume there won't be a user impact if we later change from Lerna to npm, so starting with Lerna sounds quite reasonable.

Reply:
Yeah, the `npm publish` currently seems workspaces-unaware and there is no immediate plan to fix it: https://github.com/npm/rfcs/pull/117/files#diff-f2d6f6a359bc25897c95aa08add740198c6b7743b5f2f5a9456535c5d7244fbdR71

Personally I think it's unusable without the proper publish command 🤔

Reply:
Lerna it is then :)